public/app/plugins/datasource/tempo/QueryEditor/NativeSearch.test.tsx

```diff
@@ -56,7 +57,7 @@ describe('NativeSearch', () => {
       <NativeSearch datasource={{} as TempoDatasource} query={mockQuery} onChange={jest.fn()} onRunQuery={jest.fn()} />
     );
 
-    const asyncServiceSelect = screen.getByRole('combobox', { name: 'select-span-name' });
+    const asyncServiceSelect = screen.getByRole('combobox', { name: 'select-service-name' });
```

Comment:
I don't know how nit-picky we are in PRs, but the `select` isn't anymore a `asyncSelect`, such that this variablename might be outdated. But considering it is only in tests, it does not matter too much.